products/dotnet/get-started.adoc

```diff
@@ -35,17 +35,21 @@ Finally, yes, you can install on bare metal if you wish. #{site.base_url}/produc
 
 Here’s the list of instructions to install .NET Core on Red Hat Enterprise Linux:
 
-1. Run `sudo yum install -y rh-dotnetcore10`
+1. Register your RHEL machine using your developer subscription
+2. Run `subscription-manager repos --enable=rhel-7-server-dotnet-rpms`
```

Comment:
@rctrh hey, is there a fast way to preview my changes? I have changed the .NET docs to follow RHSCL docs closely (https://github.com/omajid/developers.redhat.com/commit/64aaa4fc36ad4d751bfb9cc7c39a0ce661a6e33c), but I would like to preview my changes before opening another PR.